starters/features/leaflet-map/package.json

```
@@ -0,0 +1,30 @@
+{
+  "description": "Use Leaflet Maps in your Qwik app",
+  "__qwik__": {
+    "displayName": "Integration: Leaflet Maps",
+    "priority": -10,
+    "viteConfig": {},
+    "docs": [
+      "https://leafletjs.com/reference.html",
+      "https://leafletjs.com/examples/quick-start/",
+      "https://www.sitepoint.com/leaflet-create-map-beginner-guide/"
+    ],
+    "nextSteps": {
+      "lines": [
+        "We start the project with `yarn start`.",
```

Modified like this it is tool agnostic
```suggestion
        "We start the project with the `start` npm script",
```